spec/ttml1.xml

```diff
@@ -1097,6 +1097,35 @@ conformant <emph>Document Instance</emph>, a given <emph>Document Instance</emph
 constrained by the author or authoring tool to satisfy a more
 restrictive definition of validity.</p>
 </note>
+<note>
+<p>As illustrated in the following example, an <emph>Abstract Document Instance</emph> can be a <emph>Valid Abstract Document Instance</emph> even if it includes elements and attributes whose namespace names are listed in <specref ref="namespace-vocab-table"/> but are not defined in this version of the specification. Specifically, the element <code>foo</code> and the attribute <att>tts:foo</att> are pruned by above steps (1) and (3), respectively, because they are not members of the associated <emph>Abstract Document Type</emph>, even though their namespace names are listed in <specref ref="namespace-vocab-table"/>.
+</p>
+<table id="document-types-example" role="example">
+<caption>Example Fragment &ndash; Pruning of elements and attributes that are not members of the associated <emph>Abstract Document Type</emph></caption>
+<tbody>
+<tr>
+<td>
+<eg xml:space="preserve">
+&lt;tt
+  xmlns=&quot;http://www.w3.org/ns/ttml&quot;
+  xmlns:tts=&quot;http://www.w3.org/ns/ttml#styling&quot;
+  xml:lang=&quot;en&quot;&gt;
+  &lt;body&gt;
+    &lt;foo&gt;Foo&lt;/foo&gt;
+    &lt;div&gt;
+      &lt;p
+        tts:foo=&quot;bar&quot;&gt;
+          Bar
+      &lt;/p&gt;
+    &lt;/div&gt;
+  &lt;/body&gt;
+&lt;/tt&gt;
+</eg>
+</td>
+</tr>
+</tbody>
+</table>
```

Comment:
It seems to be more common in the spec to explain the example in following text rather than preceding text. Suggest moving the explanation to here.